org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/SonarLintImages.java

```diff
@@ -97,16 +101,20 @@ public static Image getTypeImage(String type) {
     private final ImageDescriptor severity;
     private final ImageDescriptor type;
 
-    public CompositeSeverityTypeImage(ImageDescriptor severity, ImageDescriptor type) {
+    public CompositeSeverityTypeImage(ImageDescriptor severity, @Nullable ImageDescriptor type) {
       this.severity = severity;
       this.type = type;
     }
 
     @Override
     protected void drawCompositeImage(int width, int height) {
       // Keep using deprecated methods for backward compatibility
-      drawImage(type.getImageData(), 0, 0);
-      drawImage(severity.getImageData(), 16, 0);
+      if (type != null) {
+        drawImage(type.getImageData(), 0, 0);
+        drawImage(severity.getImageData(), 16, 0);
+      } else {
+        drawImage(severity.getImageData(), 0, 0);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "drawImage"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "getImageData"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)